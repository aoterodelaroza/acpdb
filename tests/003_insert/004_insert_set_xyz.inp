## check: 004_insert_set_xyz.out -a1e-10
## delete: 004_insert_set_xyz.db
## labels: regression quick

system rm -f 004_insert_set_xyz.db
connect 004_insert_set_xyz.db

insert litref bibtex ../zz_source/bsip2.bib
insert set s22x5
 property_type energy_difference
 description A description.
 litrefs bsip1 boys1970
 xyz ../zz_source/s22_xyz/ch4_ch4.xyz ../zz_source/s22_xyz/ch4_ch4_1.xyz \
     ../zz_source/s22_xyz/ch4_ch4_2.xyz
end
