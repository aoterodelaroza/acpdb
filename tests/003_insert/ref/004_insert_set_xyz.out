* SYSTEM: rm -f 004_insert_set_xyz.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 004_insert_set_xyz.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)

* INSERT: insert data into the database (SET)

